openmetadata-ui/src/main/resources/ui/src/styles/variables.less

```diff
@@ -24,7 +26,7 @@
 @border-radius-base: 2px;
 @box-shadow-base: 0 3px 6px -4px rgba(0, 0, 0, 0.12),
   0 6px 16px 0 rgba(0, 0, 0, 0.08), 0 9px 28px 8px rgba(0, 0, 0, 0.05);
-@table-header-bg: #fff;
+@white: #fff;
```

Comment:
You must be having this variable from AntD

Reply:
I did try but to use antD variable I need to import `@import '~antd/dist/antd.less';` and which causes to override other styling